shared/css/top/top.css

```diff
@@ -38,11 +38,61 @@ body {
   word-wrap: break-word;
 }
 
+#post_content {
+    background: white;
+    border-radius: 6px;
+    padding: 13px;
+    box-shadow: 4px 5px 0px #8E8980;
+}
+
+#post_content p {
+  margin: 0;
+  word-wrap: break-word;
+}
+
+#cleartext {
+  height: 100%;
+}
+
+@media(min-width: 992px){
+  #dashed_hr {
```

Comment:
That is hackier (on Bootstrap's account) then I would have expected since different devices have different pixel densities. Good to know, thanks.